middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -822,12 +861,14 @@ let after_downwards_traversal_of_one_recursive_let_cont_handler cont
     DA.map_data_flow dacc ~f:(fun data_flow ->
         Data_flow.add_extra_params_and_args cont extra_params_and_args data_flow)
   in
-  let cont_uses_env = CUE.remove (DA.continuation_uses_env dacc) cont in
+  let cont_uses_env_after_body = DA.continuation_uses_env dacc in
```

Comment:
Do we need to keep all of the continuation uses env ? Or could we do with just the uses of the continuation being simplified ?